MonoGame.Framework/Graphics/GraphicsMetrics.cs

```diff
@@ -0,0 +1,64 @@
+﻿// MonoGame - Copyright (C) The MonoGame Team
+// This file is subject to the terms and conditions defined in
+// file 'LICENSE.txt', which is part of this source code package.
+
+namespace Microsoft.Xna.Framework.Graphics
+{
+    /// <summary>
+    /// A snapshot of rendering statistics from <see cref="GraphicsDevice.Metrics"/> to be used for runtime debugging and profiling.
+    /// The metrics are reset every frame after draw within <see cref="GraphicsDevice.Present"/>. 
```

Comment:
Can you move this...
```
The metrics are reset every frame after draw within <see cref="GraphicsDevice.Present"/>. 
```
... to the docs for `GraphicsDevice.Metrics`.

How you have this now it implies that I every instance of `GraphicsMetrics` somehow magically resets on `GraphicsDevice.Present()` which actually only `GraphicsDevice.Metrics` resets like this.